<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerMenu</class>
 <widget class="QMainWindow" name="CustomerMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scooter app - customer menu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbSearch">
        <property name="text">
         <string>Search scooters by location</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbReserve">
        <property name="text">
         <string>Reserve a scooter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbAllReserved">
        <property name="text">
         <string>View reserved scooters</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbUse">
        <property name="text">
         <string>Use a scooter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbAllInUse">
        <property name="text">
         <string>View in-use scooters</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbFilter">
        <property name="text">
         <string>Filter scooters</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QPushButton" name="pbMileage">
          <property name="text">
           <string>By Mileage</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbAge">
          <property name="text">
           <string>By Age</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pbPark">
        <property name="text">
         <string>Park a scooter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbReturn">
        <property name="text">
         <string>Return to user choice</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lScooterList">
        <property name="text">
         <string>Scooter List:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lReservedScooters">
        <property name="text">
         <string>Reserved Scooters:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lInUseScooters">
        <property name="text">
         <string>In-Use Scooters:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listViewAll"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lInput">
          <property name="text">
           <string>Input:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leInput"/>
        </item>
        <item>
         <widget class="QPushButton" name="pbSearchGo">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lReserve">
          <property name="text">
           <string>Reserve A Scooter:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lUse">
          <property name="text">
           <string>Use A Scooter:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lPark">
          <property name="text">
           <string>Park A Scooter:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="lwParkedScootes"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="lEnterID">
            <property name="text">
             <string>Enter ID:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leEnterID"/>
          </item>
          <item>
           <widget class="QPushButton" name="pbGo">
            <property name="text">
             <string>Go</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
